frontend/src/components/Forms/reports/ReportFileUpload.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Field } from "redux-form";
+import { Form } from "antd";
+import { MINE_INCIDENT } from "@/constants/API";
+import FileUpload from "@/components/common/FileUpload";
+import { DOCUMENT, EXCEL } from "@/constants/fileTypes";
+
+const propTypes = {
+  onFileLoad: PropTypes.func.isRequired,
+  onRemoveFile: PropTypes.func.isRequired,
+  mineGuid: PropTypes.string.isRequired,
+  allowMultiple: PropTypes.bool.isRequired,
+};
+
+export const ReportFileUpload = (props) => (
+  <Form.Item>
+    <Field
+      id="fileUpload"
+      name="fileUpload"
+      component={FileUpload}
+      uploadUrl={`${MINE_INCIDENT()}/documents?mine_guid=${props.mineGuid}`}
```

Comment:
This url looks weird. should it be `mine/<guid>/reports/documents` which should become a constant in the API file.

Reply:
That looks weird because it shouldn't be here that component was going to be used but was never used. I have removed the whole component.